pkgs/top-level/all-packages.nix

```diff
@@ -16339,20 +16339,20 @@ with pkgs;
     inherit (darwin) apple_sdk;
   };
 
-  rust_1_69 = callPackage ../development/compilers/rust/1_69.nix {
+  rust_1_70 = callPackage ../development/compilers/rust/1_70.nix {
     inherit (darwin.apple_sdk.frameworks) CoreFoundation Security SystemConfiguration;
     llvm_15 = llvmPackages_15.libllvm;
```

Comment:
llvm should be updated to 16, see https://github.com/rust-lang/rust/releases/tag/1.70.0 (internal changes)